picard/ui/cdlookup.py

```diff
@@ -87,7 +88,8 @@ def myjoin(values):
                 item.setText(4, myjoin(labels))
                 item.setText(5, myjoin(catalog_numbers))
                 item.setText(6, barcode)
-                item.setText(7, release.get('disambiguation', ''))
+                item.setText(7, medium.get('format', '') if medium else '')
+                item.setText(8, release.get('disambiguation', ''))
```

Comment:
Not directly related, but we should perhaps generalize the use of https://github.com/metabrainz/picard/blob/8e95570376f271e97c62c0aa507b888f76826dc7/picard/ui/itemviews/columns.py 
Having numeric indexes for columns isn't great.

Reply:
Yes, seems sensible

Reply:
See https://github.com/metabrainz/picard/pull/2615

Reply:
Done, updated the code in the last commit